pages/advanced/cli-reference/unattended-commands.mdx

```diff
@@ -0,0 +1,132 @@
+```
+
+### Send ERC-20 tokens
+
+Sends an ERC-20 token from the Safe account to a different one.
+
+```bash
+safe-cli send-erc20 [OPTIONS] SAFE_ADDRESS NODE_URL TO TOKEN_ADDRESS AMOUNT
+
+╭─ Arguments ────────────────────────────────────────────────────────────────────────────────╮
+│ *    safe_address      CHECKSUMADDRESS  The address of the Safe. [required]                |
+│ *    node_url          TEXT             Ethereum node url. [required]                      │
+│ *    to                CHECKSUMADDRESS  The address of destination. [required]             │
+│ *    token_address      CHECKSUMADDRESS  Erc20 token address. [required]                   │
+│ *    amount             INTEGER          Amount of erc20 tokens in wei to send. [required] │
+╰────────────────────────────────────────────────────────────────────────────────────────────╯
+╭─ Options ──────────────────────────────────────────────────────────────────────────────────╮
+│ --private-key                         TEXT     List of private keys of signers.            │
+│ --safe-nonce                          INTEGER  Force nonce for tx_sender                   │
+│ --interactive    --non-interactive             Enable / disable interactive mode.          │
+╰────────────────────────────────────────────────────────────────────────────────────────────╯
+```
+
+### Send ERC-721 tokens
+
+Sends an ERC-721 token from the Safe account to a different one.
+
+```bash
+safe-cli send-erc721 [OPTIONS] SAFE_ADDRESS NODE_URL TO TOKEN_ADDRESS TOKEN_ID
+
+╭─ Arguments ───────────────────────────────────────────────────────────────────────╮
+│ *    safe_address      CHECKSUMADDRESS  The address of the Safe. [required]       |
+│ *    node_url          TEXT             Ethereum node url. [required]             │
+│ *    to                CHECKSUMADDRESS  The address of destination. [required]    │
+│ *    token_address      CHECKSUMADDRESS  Erc20 token address. [required]          │
+│ *    token_id           INTEGER          Erc721 token id. [required]              │
+╰───────────────────────────────────────────────────────────────────────────────────╯
+╭─ Options ─────────────────────────────────────────────────────────────────────────╮
+│ --private-key                         TEXT     List of private keys of signers.   │
+│ --safe-nonce                          INTEGER  Force nonce for tx_sender          │
+│ --interactive    --non-interactive             Enable / disable interactive mode. │
+╰───────────────────────────────────────────────────────────────────────────────────╯
+```
+
+### Transaction builder
+
+Execute a transaction or transaction batch from a `JSON` file. The format of the file is the same as the one used from the [Safe\{Wallet}](https://app.safe.global) website in the `Transaction Builder` application.
+
+This [guide](https://help.safe.global/en/articles/40841-transaction-builder) explains how to use the tx-builder application.
```

Comment:
Please, DevEx Team, can you check the feasibility of including this link? There is no documentation on the JSON format supported here and this is the only reference I found to using the transaction builder APP. 
Thanks